Gemfile

```diff
@@ -2,7 +2,12 @@
 
 source 'https://rubygems.org'
 
-gem 'temporalio'
+# Primary Temporal dependency
+gem 'temporalio', path: '../temporal-sdk-ruby/temporalio'
+
+# Additional dependencies for certain samples
+gem 'google-protobuf', '~> 3.25'
+gem 'temporal-ruby', path: '../coinbase-temporal-ruby'
```

Comment:
Point to GitHub repo when https://github.com/coinbase/temporal-ruby/pull/335 merged (or consider pointing to fork temporarily)